README.md

```diff
@@ -1,46 +1,63 @@
-# Getting Started with Create React App and Redux
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+# React-Redux Capstone for Microverse
 
-## Available Scripts
+Web application that utilizes Riot's League of Legends API to display a list of character and alloq filtering through them, as well as seeing additional details for each character.
 
-In the project directory, you can run:
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- HTML/CSS/JS
+- React
+- Redux
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Quick View
 
-### `npm test`
+![Untitled](https://user-images.githubusercontent.com/10102977/195940045-5685094a-0d55-4047-978d-c72625a9edef.png)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Demo
 
-### `npm run build`
+[Live Demo Link](https://cosmic-choux-a8eea3.netlify.app/)
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Video Prresentation
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+[Video Presentation](https://www.loom.com/share/5449f045ce78471a9eef5abef2c1f6f9)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Getting Started
 
-### `npm run eject`
+### Prerequisites
+- Node.js
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+### Setup
+To get a local copy up and running follow these simple example steps.
+- Clone the repo
+- cd into the project directory
+- npm install
+- npm start (to start/run the project)
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Authors
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.